common/app/conf/PanicSheddingFilter.scala

```diff
@@ -41,40 +41,51 @@ class PanicSheddingFilter extends Filter with Logging {
     val surgeFactor = surgeFactorCounter.get.ratio
     lazy val loadAverage = ManagementFactory.getOperatingSystemMXBean.getSystemLoadAverage
 
+    def logIt(message: Option[String]) = {
```

Comment:
Would adding a default parameter value `...message: Option[String] = None...` make call without message easier to read and write: `logIt()` instead of `logIt(None)`